test/functional/wallet_balance.py

```diff
@@ -156,7 +162,12 @@ def test_balances(*, fee_node_1=0):
             expected_balances_1 = {'mine':      {'immature':          Decimal('0E-8'),
                                                  'trusted':           Decimal('0E-8'),  # node 1's send had an unsafe input
                                                  'untrusted_pending': Decimal('30.0') - fee_node_1}}  # Doesn't include output of node 0's send since it was spent
-            assert_equal(self.nodes[0].getbalances(), expected_balances_0)
+            if self.options.descriptors:
```

Comment:
In commit "Move import and watchonly tests to be legacy wallet only in wallet_balance.py" (8ee94451ad55b81a53c38c6b8cee0c13f1b1f9e7)

Would suggest writing this as:

```python
if self.options.descriptors:
   del expected_balances_0['watchonly']
assert_equal(self.nodes[0].getbalances(), expected_balances_0)
```

to keep checking for new fields as strict and to duplicate less code

Reply:
Done